packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -35,9 +35,9 @@ For concurrent rate limits, traffic is measured in three different areas. Counts
 * For Office365 traffic, the limit is 75 concurrent transactions per org.
 * For all other traffic, including API requests, the limit is described in the table below.
 
-| Developer (free) | Developer (paid) | One App | Enterprise | Workforce Identity |
-| ---------------- | ---------------- | ------- | ---------- | ------------------ |
-| 15               | 35               | 35      | 75         | 75                 |
+| Developer (free)  | Developer (paid)  | One App  | Enterprise  | Workforce Identity  |
```

Comment:
nit: curious why an extra space added here?

Reply:
that was just me being OCD and making it readable in the markdown. Doesn't affect the live display of the table.